src/Compilers/VisualBasic/Portable/Symbols/EventSymbol.vb

```diff
@@ -137,6 +137,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End Get
         End Property
 
+        Private ReadOnly Property IEventSymbol_Parameters As ImmutableArray(Of IParameterSymbol) Implements IEventSymbol.Parameters
+            Get
+                Return ImmutableArray(Of IParameterSymbol).CastUp(Me.DelegateParameters)
```

Comment:
Ok.  I have a potential fix.  Specifically, i only return the parameters if all the parameters' '.ContainingSymbol' is the event symbol.  This means that if we pulled the parameters off of the Delegate's Invoke method, then we won't return anything.  This should mean that we only get parameters for the special VB form.

Let me know where i should add tests for this so we can validate this behavior.  Thanks!